tests/unit/entity_id_helper_test.py

```
@@ -117,3 +122,90 @@ def test_parse_and_format_without_checksum():
+        result = perform_query_to_mirror_node("http://mirror-node/accounts/123")
+        assert result == {"account": "0.0.777"}
+
+def test_perform_query_to_mirror_node_failure():
+    """Test mirror node failure handling."""
+    
+    with patch("hiero_sdk_python.utils.entity_id_helper.requests.get") as mock_get:
+        mock_get.side_effect = requests.RequestException("boom")
+
+        try:
+            perform_query_to_mirror_node("http://mirror-node/accounts/123")
+            assert False, "Should have raised RuntimeError"
```

Ruff (0.14.10)

Assertion always fails, replace with `pytest.fail()`

(PT015)